app/controllers/deidentified_clients_controller.rb

```diff
@@ -20,6 +20,7 @@ def create
   end
 
   def update
+    add_shelter_history(clean_params(deidentified_client_params))
```

Comment:
It looks to me like with pushing the client params and clients_path down we could pull update up to `NonHmisClientsController`?

Reply:
I'm going to leave this now, but I agree, we should eventually clean it up.